src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
You should avoid using the '*' selector as it impacts the performance. It's better to set styles only for elements that require them.